cosmwasm/contracts/wormchain-accounting/src/contract.rs

```diff
@@ -651,3 +655,24 @@ fn query_chain_registration(
         .load(deps.storage, chain)
         .map(|address| ChainRegistrationResponse { address })
 }
+
+fn query_missing_observations(
+    deps: Deps<WormholeQuery>,
+    index: u8,
+) -> StdResult<MissingObservationsResponse> {
+    let mut missing = Vec::new();
+    for pending in PENDING_TRANSFERS.range(deps.storage, None, None, Order::Ascending) {
+        let (_, v) = pending?;
```

Comment:
under what circumstances would `pending` be an error here? would it make sense to just ignore it instead of failing the whole query?

Reply:
It would be an error if it failed to deserialize the key or value from the underlying data, which should only happen if there is corrupted data in the store.

Data corruption implies bigger issues so I don't know if we should just silently ignore it.  That said, I don't think we'll actually hit this error in practice.

Reply:
that makes sense, thanks